Add CPF generator

rohlacanna opened the pull request:
I've added:

CPF Generator (Individual Identification Register) of Brazil.

Ref: https://en.wikipedia.org/wiki/CPF_number

- [x] USAGE.md docs if applicable
- [x] CHANGELOG.md

rohlacanna committed:
Add CPF generator

rohlacanna committed:
Update changelog and usage

caike commented:
Nice! This will be very helpful 🙏

rohlacanna committed:
Merge branch 'master' into feature/cpf-generator

arielmdc commented:
Very good!

LukeberryPi commented:
up!

joeljuca commented:
Same as #571 – the pipeline is dead (the Credo job seems to be screwing things up somehow).

rohlacanna commented:
> Same as #571 – the pipeline is dead (the Credo job seems to be screwing things up somehow).

Hey! Not sure I understood completely.. How can I help fix the pipeline issue with the Credo job? 

cc: @joeljuca

joeljuca commented:
> _Hey! Not sure I understood completely.. How can I help fix the pipeline issue with the Credo job?_

Hi @rohlacanna, it's been some weeks since I commented here, but you can see there's a dead Credo job that was holding the pipeline – since it's not running, we're unable to know if the build passes for this PR or not.

I can't tell if it was a Credo's or a GitHub Actions' issue, but you can see it in the job status:

![credo Expected — Waiting for status to be reported](https://github.com/user-attachments/assets/94b67673-8bb7-4b19-8dc0-fe7ef09f8253 "credo Expected — Waiting for status to be reported")

igas committed:
feat: add release automation (#581)

renovate[bot] committed:
chore(deps): update dependency erlang to v27.2.1 (#583)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

igas committed:
fix(deps): add missing @semantic-release/exec (#584)

igas committed:
fix(ci): typo in workflow path (#585)

igas committed:
fix(ci): add workflow_call (#586)

renovate[bot] committed:
chore(deps): update actions/setup-node action to v4.2.0 (#588)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update node.js to v22.13.1 (#587)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency elixir to v1.18.2 (#589)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

igas committed:
fix(ci): allow write (#590)

igas committed:
fix(ci): permissions (#591)

semantic-release-bot committed:
chore(release): v0.19.0-alpha.1 [skip ci]

## [0.19.0-alpha.1](https://github.com/elixirs/faker/compare/v0.18.0...v0.19.0-alpha.1) (2025-01-31)

### Features

* add release automation ([#581](https://github.com/elixirs/faker/issues/581)) ([0d08ed2](https://github.com/elixirs/faker/commit/0d08ed231c742b247e692f0b9a3773a4ce2e172b))

### Bug Fixes

* **deps:** add missing @semantic-release/exec ([#584](https://github.com/elixirs/faker/issues/584)) ([bfea5c4](https://github.com/elixirs/faker/commit/bfea5c430bb518e04fd1d508266ccb2c72c07cfd))
* **ci:** add workflow_call ([#586](https://github.com/elixirs/faker/issues/586)) ([c25f43e](https://github.com/elixirs/faker/commit/c25f43efa234d349a6adab22e60df58fcaf61db1))
* **ci:** allow write ([#590](https://github.com/elixirs/faker/issues/590)) ([d49419a](https://github.com/elixirs/faker/commit/d49419acdb81228ce0a5cfdfa8120fa427a892ad))
* elixir 17 & 18 compatibility ([#565](https://github.com/elixirs/faker/issues/565)) ([f3d4808](https://github.com/elixirs/faker/commit/f3d480885b12fba632ff21cb3f3b74845c362cb4))
* elixir deprecations ([#571](https://github.com/elixirs/faker/issues/571)) ([b7e4af0](https://github.com/elixirs/faker/commit/b7e4af033b3d5e1630cf642af96c5d7c03100c6f))
* **ci:** permissions ([#591](https://github.com/elixirs/faker/issues/591)) ([d78c8f1](https://github.com/elixirs/faker/commit/d78c8f1abf683cf9b36908338b20e7dd064fe330))
* remove nil option for Airports IATA ([#492](https://github.com/elixirs/faker/issues/492)) ([59951ec](https://github.com/elixirs/faker/commit/59951ec3a72babae936fff22883b5e821971b8b6))
* **ci:** typo in workflow path ([#585](https://github.com/elixirs/faker/issues/585)) ([8ed49eb](https://github.com/elixirs/faker/commit/8ed49eba4096eac5b12b6913e4775cb2f69785c4))

igas committed:
fix: makeup packages (#592)

igas committed:
Revert "chore(release): v0.19.0-alpha.1 [skip ci]"

This reverts commit af4f497fc9c21fa5adf623accc2b513f6d9effe0.

igas committed:
fix: docs generation (#594)

semantic-release-bot committed:
chore(release): v0.19.0-alpha.1 [skip ci]

## [0.19.0-alpha.1](https://github.com/elixirs/faker/compare/v0.18.0...v0.19.0-alpha.1) (2025-02-01)

### Features

* add release automation ([#581](https://github.com/elixirs/faker/issues/581)) ([0d08ed2](https://github.com/elixirs/faker/commit/0d08ed231c742b247e692f0b9a3773a4ce2e172b))

### Bug Fixes

* **deps:** add missing @semantic-release/exec ([#584](https://github.com/elixirs/faker/issues/584)) ([bfea5c4](https://github.com/elixirs/faker/commit/bfea5c430bb518e04fd1d508266ccb2c72c07cfd))
* **ci:** add workflow_call ([#586](https://github.com/elixirs/faker/issues/586)) ([c25f43e](https://github.com/elixirs/faker/commit/c25f43efa234d349a6adab22e60df58fcaf61db1))
* **ci:** allow write ([#590](https://github.com/elixirs/faker/issues/590)) ([d49419a](https://github.com/elixirs/faker/commit/d49419acdb81228ce0a5cfdfa8120fa427a892ad))
* docs generation ([#594](https://github.com/elixirs/faker/issues/594)) ([572dfb4](https://github.com/elixirs/faker/commit/572dfb4060a7d2a266da2edc3bde9111a8bd6c66))
* elixir 17 & 18 compatibility ([#565](https://github.com/elixirs/faker/issues/565)) ([f3d4808](https://github.com/elixirs/faker/commit/f3d480885b12fba632ff21cb3f3b74845c362cb4))
* elixir deprecations ([#571](https://github.com/elixirs/faker/issues/571)) ([b7e4af0](https://github.com/elixirs/faker/commit/b7e4af033b3d5e1630cf642af96c5d7c03100c6f))
* makeup packages ([#592](https://github.com/elixirs/faker/issues/592)) ([3f8cbc1](https://github.com/elixirs/faker/commit/3f8cbc143184344d09ef53e1740eff0c49379181))
* **ci:** permissions ([#591](https://github.com/elixirs/faker/issues/591)) ([d78c8f1](https://github.com/elixirs/faker/commit/d78c8f1abf683cf9b36908338b20e7dd064fe330))
* remove nil option for Airports IATA ([#492](https://github.com/elixirs/faker/issues/492)) ([59951ec](https://github.com/elixirs/faker/commit/59951ec3a72babae936fff22883b5e821971b8b6))
* **ci:** typo in workflow path ([#585](https://github.com/elixirs/faker/issues/585)) ([8ed49eb](https://github.com/elixirs/faker/commit/8ed49eba4096eac5b12b6913e4775cb2f69785c4))

renovate[bot] committed:
chore(deps): update dependency @semantic-release/exec to v7 (#593)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency @semantic-release/exec to v7.0.2 (#595)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency @semantic-release/exec to v7.0.3 (#597)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency erlang to v27.2.2 (#601)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency ex_doc to == 0.37.0 (#598)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): lock file maintenance (#596)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

rohlacanna committed:
Merge branch 'master' into feature/cpf-generator

rohlacanna committed:
Merge branch 'elixirs:dev' into feature/cpf-generator

rohlacanna commented:
> > _Hey! Not sure I understood completely.. How can I help fix the pipeline issue with the Credo job?_
> 
> Hi @rohlacanna, it's been some weeks since I commented here, but you can see there's a dead Credo job that was holding the pipeline – since it's not running, we're unable to know if the build passes for this PR or not.
> 
> I can't tell if it was a Credo's or a GitHub Actions' issue, but you can see it in the job status:
> 
> ![credo Expected — Waiting for status to be reported](https://private-user-images.githubusercontent.com/673884/406753864-94b67673-8bb7-4b19-8dc0-fe7ef09f8253.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxNjU4MTcsIm5iZiI6MTczOTE2NTUxNywicGF0aCI6Ii82NzM4ODQvNDA2NzUzODY0LTk0YjY3NjczLThiYjctNGIxOS04ZGMwLWZlN2VmMDlmODI1My5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxMFQwNTMxNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZDNjMzBkMGJhYjIxMTY1YzkxMDdlZjE4ZDBiNzVlMTQ1OTU3YjAwNmY2ZTIzZTFlYWZiMDRjYzAxOWNiNGUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5MkHJmOiAIOX48jmb2_shJoelBxTf2NmhwEp2wx7Ktg)

Got it, my friend. Thx! I think everything’s all good now.

rohlacanna commented:
Hey, can you check if everything's good for approval, pls?

cc: @igas

rohlacanna commented:
Hey, can you check if everything's good for approval, pls?

cc: @igas

rohlacanna commented:
Hey, can you check if everything's good for approval, pls?

cc: @igas

rohlacanna commented:
Hey, can you check if everything's good for approval, pls?

cc: @igas
